File: CoWorkHub/UI/coworkhub_desktop/lib/models/working_space_image.dart
import 'package:json_annotation/json_annotation.dart';

part 'working_space_image.g.dart';

@JsonSerializable()
class WorkingSpaceImage {
  int? imageId;
  int? workingSpacesId;
  String? imagePath;
  bool? isDeleted;

  WorkingSpaceImage({
    this.imageId,
    this.workingSpacesId,
    this.imagePath,
    this.isDeleted,
  });

  factory WorkingSpaceImage.fromJson(Map<String, dynamic> json) =>
      _$WorkingSpaceImageFromJson(json);

  Map<String, dynamic> toJson() => _$WorkingSpaceImageToJson(this);
}

File: CoWorkHub/CoWorkHub.Model/WorkingSpaceImage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CoWorkHub.Model
{
    public class WorkingSpaceImage
    {
        public int ImageId { get; set; }
        public int WorkingSpacesId { get; set; }
        public string ImagePath { get; set; } = null!;
        public bool IsDeleted { get; set; } = false;
    }
}
